handlers/viber/viber.go

```diff
@@ -96,7 +105,28 @@ func (h *handler) receiveEvent(ctx context.Context, channel courier.Channel, w h
 		return nil, handlers.WriteAndLogRequestIgnored(ctx, h, channel, w, r, "webhook valid")
 
 	case "conversation_started":
-		return nil, handlers.WriteAndLogRequestIgnored(ctx, h, channel, w, r, "ignored conversation start")
+		msgText := channel.StringConfigForKey(configViberWelcomeMessage, "")
+		if msgText == "" {
+			return nil, handlers.WriteAndLogRequestIgnored(ctx, h, channel, w, r, "ignored conversation start")
+		}
+
+		viberID := payload.User.ID
+		ContactName := payload.User.Name
+
+		// build the URN
+		urn, err := urns.NewURNFromParts(urns.ViberScheme, viberID, "", "")
+		if err != nil {
+			return nil, handlers.WriteAndLogRequestIgnored(ctx, h, channel, w, r, "ignored conversation start")
```

Comment:
Should log this as an error as well, shouldn't be a normal case where this wouldn't be valid.